flutter/shell/platform/tizen/channels/keyboard_channel.cc

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "keyboard_channel.h"
+
+#include "flutter/shell/platform/common/client_wrapper/include/flutter/standard_method_codec.h"
+#include "flutter/shell/platform/tizen/logger.h"
+
+namespace flutter {
+
+namespace {
+
+constexpr char kChannelName[] = "flutter/keyboard";
+constexpr char kGetKeyboardStateMethod[] = "getKeyboardState";
+
+}  // namespace
+
+KeyboardChannel::KeyboardChannel(BinaryMessenger* messenger) {
+  channel_ = std::make_unique<MethodChannel<EncodableValue>>(
+      messenger, kChannelName, &StandardMethodCodec::GetInstance());
+  channel_->SetMethodCallHandler(
+      [this](const MethodCall<EncodableValue>& call,
+             std::unique_ptr<MethodResult<EncodableValue>> result) {
+        HandleMethodCall(call, std::move(result));
+      });
+}
+
+KeyboardChannel::~KeyboardChannel() {}
+
+void KeyboardChannel::HandleMethodCall(
+    const MethodCall<EncodableValue>& method_call,
+    std::unique_ptr<MethodResult<EncodableValue>> result) {
+  const std::string& method_name = method_call.method_name();
+  if (method_name == kGetKeyboardStateMethod) {
+    if (!pressing_records_.empty()) {
+      EncodableMap map;
+      for (const auto& key : pressing_records_) {
+        EncodableValue physical_value(static_cast<int64_t>(key.first));
+        EncodableValue logical_value(static_cast<int64_t>(key.second));
+        map[physical_value] = logical_value;
+      }
+
+      result->Success(EncodableValue(map));
+      return;
```

Comment:
`return` may be omitted for consistency.
```suggestion
      result->Success(EncodableValue(map));
```